functions.php

```diff
@@ -16,14 +18,34 @@ function displayBook(array $book): string
     if(array_key_exists('name', $book) && array_key_exists('author', $book) && array_key_exists('category', $book) && array_key_exists('released', $book)) {
         return '<div class="item">' .
             '<div class="bookDetails">' .
-            '<p>' . $book['name'] . '</p>' .
-            '<p>' . $book['author'] . '</p>' .
-            '<p>' . $book['category'] . '</p>' .
-            '<p>' . $book['released'] . '</p>' .
+            '<p class="bookHeading">' . $book['name'] . '</p>' .
+            '<p class="bookAuthor">' . $book['author'] . '</p>' .
+            '<p class="bookCategory">' . $book['category'] . '</p>' .
+            '<p class="releaseYear">' . $book['released'] . '</p>' .
             '</div>' .
+            '<form method="POST"><input type="hidden" name="bookToBeDeleted" value="'.$book['name'].'">
+            <input type="submit" name="delete" value="Delete"/><br/>
+            </form>' .
             '</div>';
     } else {
         return '';
     }
 }
+
+function addBook(array $bookInfo, PDO $db)
+{
+    $query = $db->prepare('INSERT INTO `booksCollected` (`name`, `author`, `category`, `released`) VALUES (?, ?, ?, ?)');
+    if ($query->execute([$bookInfo['name'], $bookInfo['author'], $bookInfo['category'], $bookInfo['released']])) {
```

Comment:
no cheks that name, author, category, released are there? what if someone edits the form to remove those inputs